src/types/line.js

```diff
@@ -198,15 +198,7 @@ function drawLabel(ctx, line, chartArea) {
 }
 
 function getImageSize(size, value) {
-  if (typeof value === 'number') {
-    return value;
-  } else if (typeof value === 'string') {
-    const array = percentageRegex.exec(value);
-    if (array) {
-      return +array[1] / 100 * size;
-    }
-  }
-  return size;
+  return typeof value === 'number' ? value : typeof value === 'string' ? toPercent(value) * size : size;
```

Comment:
I think your original function body was better :)

```js
if (typeof value === 'number') {
  return value;
}
if (typeof value === 'string' && value.endsWith('%')) {
  return parseFloat(value) / 100 * size;
}
return size;
```

or (less readable, but slightly smaller footprint due to single `return`)

```js
function getImageSize(size, value) {
	return typeof value === 'number' ? value
		: (typeof value === 'string' && value.endsWith('%')) ? parseFloat(value) / 100 * size
			: size;
}
```

Reply:
I have changed for that reason, to reduce the size but I can change it quickly

Reply:
done